src/models/user.js

```diff
@@ -1,34 +1,27 @@
-/* eslint-disable linebreak-style */
 const mongoose = require('mongoose');
 
-UserSchema = new mongoose.Schema(
+const { Schema } = mongoose;
 
-{
-  Name:{
+const UserSchema = new Schema({
+  email: {
     type: String,
-    required: true
+    required: true,
+    trim: true
   },
-  email:{
+  password: {
     type: String,
     required: true
   },
-  city:{
-    type: String,
-    required: true
-  },
-  phone:{
-    type: String,
-    required:true
-  },
-  bloodgroup:{
+  role: {
     type: String,
-    required: true
+    default: 'basic',
```

Comment:
Is this supposed to be basic or donor?
Because basic is not in the enum options

Reply:
It's donor. Will correct it